### PR TITLE
Bugfix/theosd 13887 new access log entry notification

### PR DESCRIPTION
**Pull request by wvanhaevre**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Fixed
+
+- Conviva
+  - Updated deprecated name for acces log notification to fix broken bitrate reporting.
```
Comment (Danesz) [inline CHANGELOG.md:13]: nit-pick: typo `access`

File: Code/Conviva/Source/Events/Observers/AppEventForwarder.swift [hunk 2]
```diff
@@ -11,7 +11,7 @@ import AVFoundation
 
 fileprivate let willEnterForeground = UIApplication.willEnterForegroundNotification
 fileprivate let didEnterBackground = UIApplication.didEnterBackgroundNotification
-fileprivate let newAccessLogEntry = NSNotification.Name.AVPlayerItemNewAccessLogEntry
+fileprivate let newAccessLogEntry = AVPlayerItem.newAccessLogEntryNotification
```
Comment (Danesz) [inline Code/Conviva/Source/Events/Observers/AppEventForwarder.swift:14]: what behavior change did this cause that makes reporting correct? (e.g. did you get the other event too late? is the underlying notification name different from the other? old event is not firing due to it was removed?)

**Review by Danesz — APPROVED** [review]

File: Code/Conviva/Source/Events/Observers/AppEventForwarder.swift (hunk 2 above)
Comment (wvanhaevre) [reply Code/Conviva/Source/Events/Observers/AppEventForwarder.swift:14]: The other name was deprecated from iOS 17.2, so access log notifications were broken, so bitrate reporting was broken

File: CHANGELOG.md (hunk 1 above)
Comment (wvanhaevre) [reply CHANGELOG.md:13]: will fix

File: Code/Conviva/Source/Events/Observers/AppEventForwarder.swift (hunk 2 above)
Comment (wvanhaevre) [reply Code/Conviva/Source/Events/Observers/AppEventForwarder.swift:14]: https://developer.apple.com/documentation/foundation/nsnotification/name/4108588-avplayeritemnewaccesslogentry

**Pull request merged**